org.eclipse.xtext.xbase.testlanguages/src-gen/org/eclipse/xtext/xbase/testlanguages/bug462047/AbstractBug462047LangRuntimeModule.java

```diff
@@ -230,9 +228,4 @@ public void configureRewritableImportSectionEnablement(Binder binder) {
 		return DerivedStateAwareResourceValidator.class;
 	}
 	
-	// contributed by org.eclipse.xtext.xtext.generator.xbase.XbaseGeneratorFragment2
-	public Class<? extends IJvmModelInferrer> bindIJvmModelInferrer() {
```

Comment:
The bindings is no longer created by the fragment?

Reply:
yes. this is what https://github.com/eclipse/xtext/issues/1699 is for